unlock-app/src/components/interface/pagination/PageNumbers.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { PageGroup, PageNumber, PageNumberActive } from './Pagination'
+
+const PageNumbers = ({ numberOfPages, currentPage, goToPage }) => {
+  const pageNumber = number => {
+    return number === currentPage ? (
+      <PageNumberActive onClick={() => goToPage(number)}>
+        {number}
+      </PageNumberActive>
+    ) : (
+      <PageNumber onClick={() => goToPage(number)}>{number}</PageNumber>
+    )
+  }
+
+  // If number of pages less than 10, show all 10 page numbers
+  if (numberOfPages <= 10) {
```

Comment:
Maybe best to define `MAX_NUMBER_OF_PAGES_TO_SHOW_ALL`  constant in `constants.js`?